index.html

```diff
@@ -4,9 +4,63 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" type="text/css" href="css/style.css">
-  <title>Hello Front-End</title>
+  <title>Airbnb</title>
 </head>
 <body>
-  <h1>Hello Front-End</h1>
+
+  <main>
+    <h1>Meet Guidebooks</h1>
+    <h2>Discover hundreds of local spots recommended by Airbnb hosts</h2>
+
+<div class="flex-container">
+  <div class="img">
```

Comment:
I would recommend renaming this class so it doesn't coincide with an HTML tag. It's possible that you or somebody looking at your code could mistake the selector this class with a selector for the `img` tag (and vice-versa).